tests/sentry/snuba/test_discover.py

```diff
@@ -132,40 +133,27 @@ def test_missing_project(self):
 
     def test_field_aliasing_in_selected_columns(self):
         result = discover.query(
-            selected_columns=["project.id", "user", "release"],
+            selected_columns=["project.id", "user", "user.email", "release", "timestamp.to_hour"],
             query="",
             params={"project_id": [self.project.id]},
         )
         data = result["data"]
         assert len(data) == 1
         assert data[0]["project.id"] == self.project.id
         assert data[0]["user"] == "id:99"
+        assert data[0]["user.email"] == "bruce@example.com"
         assert data[0]["release"] == "first-release"
 
-        assert len(result["meta"]) == 3
-        assert result["meta"] == {
-            "project.id": "integer",
-            "user": "string",
-            "release": "string",
-        }
-
-    def test_field_alias_with_component(self):
```

Comment:
hm, actually more investigation is required before removing this test.

Reply:
going to move the removal of this test to another PR.

Reply:
What's the reason for removing this test? This was added in https://github.com/getsentry/sentry/pull/18192

Reply:
Yes, I saw. I unremoved it in https://github.com/getsentry/sentry/pull/24387/commits/70232534cfec3865ae72cc362044de00254f1e25.

Reply:
Preferably this test shouldn't be removed as we like to ensure the user fields are consistent with the behaviour we expect. The user fields had a history of causing confusion to the end user.